src/Ledger/Conway/Foreign/HSLedger/Ratify.agda

```diff
@@ -8,7 +8,7 @@ open import Ledger.Conway.Foreign.HSLedger.Gov
 
 open import Ledger.Enact govStructure
 
-open import Data.String.Base renaming (_++_ to _+ˢ_) hiding (show; length)
+open import Data.String.Base renaming (_++_ to _+ˢ_) hiding (show; length; String)
```

Comment:
Leftover?

Reply:
Nope, it's conflicting with the `String` from `Ledger.Conway.Foreign.HSLedger.BaseTypes.String`.